evm/src/executor/inspector/cheatcodes/env.rs

```diff
@@ -503,7 +503,7 @@ pub fn apply<DB: DatabaseExt>(
                 Ok(Bytes::new())
             },
         )??,
-        HEVMCalls::GetNonce(inner) => {
+        HEVMCalls::GetNonce1(inner) => {
```

Comment:
Because of the placement of `getNonce(Wallet)` in _abi/abi/HEVM.sol_, `getNonce(address)` was renamed to `HEVMCalls::GetNonce1`